src/sequelize/migrations/20200229091456-create-institution.js

```diff
@@ -0,0 +1,47 @@
+module.exports.up = (queryInterface, DataTypes) => queryInterface.createTable('Institutions', {
+  id: {
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+    type: DataTypes.INTEGER.UNSIGNED,
+  },
+  name: {
+    allowNull: false,
+    type: DataTypes.STRING,
+  },
+  type: {
+    allowNull: false,
+    type: DataTypes.ENUM('health', 'educational', 'others'),
+    defaultValue: 'health',
+  },
+  addressId: {
+    type: DataTypes.INTEGER,
+    references: {
+      model: { tableName: 'Addresses' },
+      key: 'id',
+    },
+    allowNull: false,
+  },
+  institutionHeadId: {
```

Comment:
I have been thinking as we already have InstitutionId in the User table its enough to complete the relationship.

Reply:
worked on the feedback